src/plasma/plasma_store.cc

```diff
@@ -340,33 +239,36 @@ void return_from_get(PlasmaStoreState *store_state, GetRequest *get_req) {
    * tables if it is present there. It should only be present there if the get
    * request timed out. */
   for (ObjectID &object_id : get_req->object_ids) {
-    remove_get_request_for_object(store_state, object_id, get_req);
+    auto &get_requests = object_get_requests_[object_id];
+    get_requests.erase(
```

Comment:
This line with the `std::remove` inside of the `erase` is hard to parse. Can you add a comment explaining what is happening here? Is there a way to make it clearer? Note that it used to be a whole separate function.